conjur/controller/init_controller.py

```diff
@@ -160,5 +160,5 @@ def ensure_overwrite_file(cls, config_file):
         """
         force_overwrite = input(f"File {config_file} exists. "
                                 f"Overwrite? (Default=yes): ").strip()
-        if force_overwrite != '' and force_overwrite.lower() != 'yes':
+        if force_overwrite != '' and force_overwrite.lower() != 'yes' and force_overwrite.lower() != 'y' :
```

Comment:
Do it using a list of possible options.
```suggestion
        if force_overwrite != '' and force_overwrite.lower() in APPROVAL_WORDS:
```